tests/unit/test_uta.py

```diff
@@ -27,8 +19,8 @@ def tpm3_exon1_exon8():
         "chr": "NC_000001.11",
         "start": 154192135,
         "end": 154170399,
-        "start_exon": 1,
-        "end_exon": 8,
+        "exon_start": 1,
+        "exon_end": 8,
```

Comment:
Change in uta services too to be consistent?

Reply:
Could you clarify?

Reply:
This doesn't apply anymore now that we'll be moving this out of here. But I was just suggesting changing start/end_exon to exon_start/end in uta_services to be consisstent.